'CPU Speed'  breaks in v1.5.0

Choosing any value other than the default triggers a compile error:
```
Arduino: 1.8.14 Hourly Build 2021/04/09 02:34 (Mac OS X), Board: "Raspberry Pi Pico, 2MB (no FS), 133 MHz, Disabled, None, Pico SDK"

/Users/xxx/Library/Arduino15/packages/rp2040/hardware/rp2040/1.5.0/cores/rp2040/main.cpp: In function 'int main()':
/Users/xxx/Library/Arduino15/packages/rp2040/hardware/rp2040/1.5.0/cores/rp2040/main.cpp:57:5: error: 'set_sys_clock_khz' was not declared in this scope
   57 |     set_sys_clock_khz(F_CPU / 1000, true);
      |     ^~~~~~~~~~~~~~~~~
exit status 1
Error compiling for board Raspberry Pi Pico.
```

 main.cpp no longer includes pico/stdlib.h, putting that back in compiles as expected. Was this an unexpected effect from PR #141 ?

--- a/cores/rp2040/main.cpp
+++ b/cores/rp2040/main.cpp
@@ -20,6 +20,7 @@
 
 #include <Arduino.h>
 #include "RP2040USB.h"
+#include <pico/stdlib.h>
 #include <pico/multicore.h>
 
 RP2040 rp2040;

